lib/matplotlib/colors.py

```diff
@@ -1686,18 +1686,15 @@ def shade(self, data, cmap, norm=None, blend_mode='overlay', vmin=None,
     def shade_rgb(self, rgb, elevation, fraction=1., blend_mode='hsv',
                   vert_exag=1, dx=1, dy=1, **kwargs):
         """
-        Take the input RGB array (ny*nx*3) adjust their color values
-        to given the impression of a shaded relief map with a
-        specified light source using the elevation (ny*nx).
-        A new RGB array ((ny*nx*3)) is returned.
+        Use this light source to adjust the colors of the *rgb* input array to
+        give the impression of a shaded relief map with the given `elevation`.
 
         Parameters
         ----------
         rgb : array-like
-            An MxNx3 RGB array, assumed to be in the range of 0 to 1.
+            An (m, n, 3) RGB array, assumed to be in the range of 0 to 1.
```

Comment:
We use capital letters for dimensions https://matplotlib.org/devdocs/devel/documenting_mpl.html#id4

Reply:
don't really like it but sure, fixed.